libbeat/processors/add_kubernetes_metadata/_meta/fields.yml

```diff
@@ -70,11 +70,13 @@
             Kubernetes statefulset name
 
         - name: container.name
-          type: keyword
+          type: alias
+          path: container.name
```

Comment:
Is this intentional?

Reply:
We want `kubernetes.container.name` to point to `container.name` (same for image) so yes?

Reply:
Yes, full paths need to be configred here, I think this is correct.

Reply:
Ahh yep sorry I missed it! Thanks!!